docs/_posts/aymanechilah/2023-01-17-ocr_base_printed_v2_opt.md

```diff
@@ -0,0 +1,141 @@
+---
+layout: model
+title: Оcr base v2 optimized for printed text
+author: John Snow Labs
+name: ocr_base_printed_v2_opt
+date: 2023-01-17
+tags: [en, licensed]
+task: OCR Text Detection & Recognition
+language: en
+edition: Visual NLP 4.2.4
+spark_version: 3.2.1
+supported: true
+article_header:
+  type: cover
+use_language_switcher: "Python-Scala-Java"
+---
+
+## Description
+
+Ocr base printed model v2 optimized with ONNX for recognise printed text based on Tesseract architecture and printed datasets.
+Tesseract is an optical character recognition engine with open-source code, this is the most popular and qualitative OCR-library. OCR uses artificial intelligence for text search and its recognition on images. Tesseract is finding templates in pixels, letters, words and sentences. It uses two-step approach that calls adaptive recognition. It requires one data stage for character recognition, then the second stage to fulfil any letters, it wasn’t insured in, by letters that can match the word or sentence context.
+
+## Predicted Entities
+
+{:.btn-box}
+<button class="button button-orange" disabled>Live Demo</button>
+[Open in Colab](https://colab.research.google.com/github/JohnSnowLabs/spark-ocr-workshop/blob/master/tutorials/Cards/SparkOcrImageToTextPrinted_V2_opt.ipynb){:.button.button-orange.button-orange-trans.co.button-icon}
```

Comment:
Colab notebook link is broken. Could you please check?

Reply:
solved